Libraries/LibELF/ELFDynamicObject.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include <LibELF/ELFImage.h>
+#include <LibELF/exec_elf.h>
+#include <dlfcn.h>
+#include <mman.h>
+
+#include <AK/OwnPtr.h>
+#include <AK/RefCounted.h>
+#include <AK/String.h>
+
+#define ALIGN_ROUND_UP(x, align) ((((size_t)(x)) + align - 1) & (~(align - 1)))
+
+class ELFDynamicObject : public RefCounted<ELFDynamicObject> {
+public:
+    static RefPtr<ELFDynamicObject> make(const char* filename, int fd, size_t file_size);
+
+    ~ELFDynamicObject();
+
+    inline bool is_valid() const { return m_valid; }
```

Comment:
No need to put `inline` here, since the definition is in-line, it's well, by definition inline. :)

Reply:
Fair enough. no inline. well, not no_inline, but not marked inline anymore.